backend/src/logging/lifecycle.ts

```
@@ -36,21 +36,25 @@ export async function initializeLogging(): Promise<void> {
 
     // Initialize each enabled driver
-    for (const driverType of config.logDrivers) {
+    for (const driverType of config.logging?.drivers ? Object.keys(config.logging.drivers) : []) {
       try {
         systemLogger.info(`Initializing log driver: ${driverType}`);
         
         switch (driverType) {
           case 'sqlite':
-            const { SqliteLogDriver } = await import('./drivers/sqlite-driver.js');
-            logManager.registerDriver(new SqliteLogDriver());
+            const { SQLiteLogDriver } = await import('./drivers/sqlite-driver.js');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)